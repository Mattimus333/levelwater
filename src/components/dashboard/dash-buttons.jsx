import React from 'react';
import DashButtonsHeader from './dash-buttons-header';
import DashButton from './dash-button';

const DashButtons = () => (
  <div id="dash-buttons">
    <DashButtonsHeader />
    <div className="ui grid">
      <div className="column eight wide"><DashButton buttonText="Recommended Rate Schedule"/></div>
      <div className="column eight wide"><DashButton buttonText="Fixed Costs"/></div>
      <div className="column eight wide"><DashButton buttonText="Projected Future Needs"/></div>
      <div className="column eight wide"><DashButton buttonText="Edit System Info"/></div>      
    </div>
  </div>
);

export default DashButtons;
