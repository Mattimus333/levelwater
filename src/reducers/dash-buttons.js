const dashButtons = (state = null, action) => {
  switch (action.type) {
  case 'SUBMIT_RATES_COSTS_BUTTON':
    return action.payload;
  default:
    return state;
  }
};

export default dashButtons;
