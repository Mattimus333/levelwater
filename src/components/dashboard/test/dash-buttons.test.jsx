import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {DashButtons} from '../dash-buttons.jsx';

describe('dash buttons', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<DashButtons />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
