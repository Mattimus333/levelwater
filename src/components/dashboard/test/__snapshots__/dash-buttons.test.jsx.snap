exports[`dash buttons renders a snapshot 1`] = `
<Route
  render={[Function]} />
`;
